README.md

```diff
@@ -35,6 +35,41 @@ These are located in the developer_scripts folder
 * `npm run devreset` runs cleandb then populate db and then starts up the server.
 * `npm run sass` watches for style changes
 
+## Docker
+This project is also dockerized (no official repository available however). This project uses and tested on the following Docker technologies:
+
+* Docker for Mac:
+   * Client version: 17.03.1-ce
+   * Server version: 17.03.1-ce
+* Docker Compose version 3
+
+### Container
+The node application is run in a container named _dockered-fish_ and the MongoDB is run in a container
```

Comment:
Is it possible to make this run in containers simply named _fish_ and _mongo-fish_?